docs/docs/explore/project-chat.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "AI Chat in Rill Cloud"
+description: Chat with your data using natural language in Rill Cloud
+sidebar_label: "AI Chat"
+sidebar_position: 05
+---
+
+## Overview
+
+AI Chat in Rill Cloud allows you to have natural language conversations with your data directly in your browser. Instead of building queries or navigating through dashboards, simply ask questions in plain English and get instant insights backed by your metrics views—complete with **interactive visualizations** and **direct links** to your existing dashboards for deeper exploration.
+
+AI Chat is powered by [Rill's Model Context Protocol (MCP)](/explore/mcp) integration, which ensures that responses are accurate, governed, and consistent with the metrics displayed in your dashboards. By querying data with **predefined measures and dimensions**, you can trust that the answers you receive are as reliable as the data in your Rill dashboards. Plus, responses include canvas dashboard components that visualize your data and automatically link back to the relevant dashboards with filters pre-applied.
+
+<img src='/img/explore/chat/project-chat.png' class='rounded-gif'/>
+<br />
+
+## How It Works
+
+AI Chat uses the same [Rill MCP Server](/explore/mcp) technology that powers external AI integrations with tools like Claude Desktop. This means:
+
+- **Governed Data Access** - The AI only queries [metrics views](/build/metrics-view) you've already defined, ensuring accuracy and consistency
+- **Structured Responses** - Results are based on predefined measures and dimensions, not raw database queries
+- **Context-Aware** - The AI understands your metrics view structure, including available dimensions, measures, and time ranges
+- **Secure** - Respects your [project's access](/build/metrics-view/security) controls and user permissions
+
+## Accessing AI Chat
+
+### Access AI Chat from Project Home
+
+1. Navigate to your [Rill Cloud](https://ui.rilldata.com) project home page
+2. Click on the **AI** tab in the project navigation
+3. Start typing your question in the chat interface
+
+### Access AI Chat from a Dashboard
```

Comment:
This is behind the `dashboardChat` feature flag, so how about we comment this out for now